OfficeApp3Web/index.html

```diff
@@ -91,7 +91,7 @@
     </div>
     <div class="toast" id="toastMessage">
         <div class="closeBtn"><img id="closeImage" src="Images/closeblack.png" role="button" tabindex="0" alt="Close" /></div>
-        <div id="message"></div>
+        <div id="message" tabindex="0" ></div>

```

Comment:
Xiaoyan's previous pull request already have this change. Please always merge from upstream branch first, then create new pull request.